src/artworks/artworks.repository.ts

```diff
@@ -4,7 +4,7 @@ import { FirebaseArtwork } from './artwork.firebase';
 import { PeopleRepository } from '../people/people.repository';
 import { withId, QueryFilter, buildQuery } from '../utils';
 import { validators } from 'validate.js';
-import firebase from 'firebase';
+import * as firebase from 'firebase/app';
```

Comment:
The Firebase module includes a default import which means we can just use `import firebase from 'firebase/app`.

Reply:
But in firebase console warning it suggest using this syntax rather than import firebase

Reply:
Oh, I see! We can keep it like this then, as long as it fixes the warning.